samples/pipeline-tests.sh

```diff
@@ -1,110 +1,196 @@
 seldon pipeline infer joincheck --inference-mode grpc     '{"model_name":"simple","inputs":[{"name":"INPUT0","contents":{"int_contents":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]},"datatype":"INT32","shape":[1,16]},{"name":"INPUT1","contents":{"int_contents":[1,2,3,4,5,6,7,8,9,10,11,12,13,14,15,16]},"datatype":"INT32","shape":[1,16]}]}'
-seldon pipeline unload joincheck
-seldon model unload tfsimple1
-seldon model unload tfsimple2
-seldon model unload tfsimple3
-seldon model unload check
+unload pipeline joincheck ./pipelines/tfsimples-join-outlier.yaml
+unload model tfsimple1 ./models/tfsimple1.yaml
+unload model tfsimple2 ./models/tfsimple2.yaml
+unload model tfsimple3 ./models/tfsimple3.yaml
+unload model check ./models/check.yaml
 
 sleep $sleepTime
 
-seldon model load -f ./models/mul10.yaml
-seldon model load -f ./models/add10.yaml
+load model ./models/mul10.yaml
+load model ./models/add10.yaml
 seldon model status mul10 -w ModelAvailable | jq -M .
 seldon model status add10 -w ModelAvailable | jq -M .
-seldon pipeline load -f ./pipelines/pipeline-inputs.yaml
+load pipeline ./pipelines/pipeline-inputs.yaml
 seldon pipeline status pipeline-inputs -w PipelineReady | jq -M .
 seldon pipeline infer pipeline-inputs --inference-mode grpc  '{"model_name":"pipeline","inputs":[{"name":"INPUT0","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]},{"name":"INPUT1","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}' | jq -M .
-seldon pipeline unload pipeline-inputs
-seldon model unload mul10
-seldon model unload add10
+unload pipeline pipeline-inputs ./pipelines/pipeline-inputs.yaml
+unload model mul10 ./models/mul10.yaml
+unload model add10 ./models/add10.yaml
 
 sleep $sleepTime
 
-seldon model load -f ./models/mul10.yaml
-seldon model load -f ./models/add10.yaml
+load model ./models/mul10.yaml
+load model ./models/add10.yaml
 seldon model status mul10 -w ModelAvailable | jq -M .
 seldon model status add10 -w ModelAvailable | jq -M .
-seldon pipeline load -f ./pipelines/trigger-joins.yaml
+load pipeline ./pipelines/trigger-joins.yaml
 seldon pipeline status trigger-joins -w PipelineReady | jq -M .
 seldon pipeline infer trigger-joins --inference-mode grpc  '{"model_name":"pipeline","inputs":[{"name":"ok1","contents":{"fp32_contents":[1]},"datatype":"FP32","shape":[1]},{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}'
 seldon pipeline infer trigger-joins --inference-mode grpc  '{"model_name":"pipeline","inputs":[{"name":"ok3","contents":{"fp32_contents":[1]},"datatype":"FP32","shape":[1]},{"name":"INPUT","contents":{"fp32_contents":[1,2,3,4]},"datatype":"FP32","shape":[4]}]}'
-seldon pipeline unload trigger-joins
-seldon model unload mul10
-seldon model unload add10
+unload pipeline trigger-joins ./pipelines/trigger-joins.yaml
+unload model mul10 ./models/mul10.yaml
+unload model add10 ./models/add10.yaml
+
+
+# MLServer
```

Comment:
Perhaps we should rename the file to "smoke-tests.sh"? (given that now we also have MLServer and Experiments)